pkg/compiler/debug.go

```diff
@@ -111,13 +111,15 @@ func (c *codegen) registerDebugVariable(name string, expr ast.Expr) {
 
 func (c *codegen) methodInfoFromScope(name string, scope *funcScope) *MethodDebugInfo {
 	ps := scope.decl.Type.Params
-	params := make([]DebugParam, 0, ps.NumFields())
-	for i := range params {
+	params := make([]DebugParam, ps.NumFields())
+	count := 0
```

Comment:
I think `append()` style would still look a bit better here (not needing any `count`, for example), to fix the original problem you only need to change the range iterator.